evap/evaluation/models.py

```diff
@@ -1497,6 +1496,49 @@ class Meta:
         verbose_name_plural = _("questions")
 
 
+class Infotext(models.Model):
+    """Infotext to display, e.g. at the student index and contributor index pages"""
+
+    title_de = models.CharField(max_length=255, verbose_name=_("title (german)"), blank=True)
+    title_en = models.CharField(max_length=255, verbose_name=_("title (english)"), blank=True)
+    title = translate(en="title_en", de="title_de", blank=True)
+
+    content_de = models.TextField(verbose_name=_("content (german)"), blank=True)
+    content_en = models.TextField(verbose_name=_("content (english)"), blank=True)
+    content = translate(en="content_en", de="content_de")
+
+    def is_empty(self):
+        return not self.title or not self.content
+
+    class LinkedPage(models.TextChoices):
+        STUDENT_INDEX = ("student_index", "Student index page")
+        CONTRIBUTOR_INDEX = ("contributor_index", "Contributor index page")
+        GRADES_PAGES = ("grades_pages", "Grade publishing pages")
+
+    linked_page = models.CharField(
```

Comment:
the name should probably be changed because the infotext is not linked to a page but placed on it. how do you like something like `position`, `placement` or just `page`?